app-cdr/bchunk/bchunk-1.2.2.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Convert CD images from bin/cue to iso+wav/cdr"
+HOMEPAGE="http://he.fi/bchunk/"
+SRC_URI="http://he.fi/bchunk/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x86-solaris"
+
+DOCS=( "${PN}.spec" README ChangeLog )
```

Comment:
I don't think adding a .spec file for us is meaningful. This whole DOCS line could be removed and just go with default `einstalldocs`.